toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -54,6 +54,31 @@ while read -r package || [ -n "$package" ]; do
     install_one_toolchain_rpm "$package"
 done < "$packages"
 
+# If the host machine rpm version is >= 4.16 (such as Mariner 2.0), it will create an "sqlite" rpm database backend incompatible with Mariner 1.0 (which uses "bdb")
```

Comment:
Anything we can do to skip this step if rpm is older (or even better if the host rpm was built with a compatible version of rpm?)

Reply:
We can probably query the rpm db backend version of the host, and skip this step. Let me take a look.

Reply:
I guess we could run `rpm -E "%{_db_backend}"` on the host and inside the chroot, skip if they match, and perform the DB conversion if they don't or the chroot run crashes (that's my guess about what would happen, if the chroot's RPM didn't support the DB created by the host's).

Reply:
If that's going to work, could we also migrate it to 2.0?:)

Reply:
Yes, I've implemented this optimization in the latest commit. I can port this to 2.0 as well.

Reply:
For 2.0 changes see https://github.com/microsoft/CBL-Mariner/pull/4025